Project 3/Task 1/diff_quantize.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+import numpy as np
+import cv2
+import sys
+from os import listdir, path
+import Utility as util
+from block_quantize import quantize, display_histogram
+
+def diff_quantization(frame_data, n):
+    diff_frame_data = np.diff(frame_data, axis=0)
+    
+    # Quantize the blocks of the video
+    quantized_values = quantize(diff_frame_data, n)
+    return quantized_values
```

Comment:
@GCrispino sorry buddy, but while looking at this code I realized we could do a massive shortcut here by creating a matrix of frame-by-frame diffs with `np.diff` and running our `quantize` function from task 1 a on that matrix, removing the need for any extra code. Could you take a look at this and let me know if you have any concerns?